pylint/checkers/stdlib.py

```diff
@@ -488,8 +488,14 @@ def __init__(self, linter: PyLinter) -> None:
         # synced with the config argument deprecated-modules
 
     def _check_bad_thread_instantiation(self, node: nodes.Call) -> None:
-        if not node.kwargs and not node.keywords and len(node.args) <= 1:
-            self.add_message("bad-thread-instantiation", node=node)
+        if (
+            len(node.args) < 2
+            and not node.kwargs
```

Comment:
Looks like a false negative here if `target` is omitted from the kwargs, like this:
```python
import threading

kw = {'target_typo': lambda x: x}
threading.Thread(None, **kw)
```
Would be good to add this as a test case.